### DISCUSSION TITLE
Dynamic className based on a cell value

### DISCUSSION THREAD

**Discussion by govi74** [TanStack/table#2256] [General]
Is it possible to dynamically generate className based on a cell value. 

The requirement is I have a severity column. Based on the severity , i need to add borderLeft of the first column with varying color based on the severity.

The class name accepts the string and how do I access the cell value to determine the column name?

Any help?

**Comment by govi74** [comment]
nvm... figured it out.. cool :)

**Reply by ayushmalviya27** [reply]
Hey can you tell me how did you do it?
I'm stuck with the same problem.
